docs/FAQ.md

```diff
@@ -636,6 +636,17 @@ If your environment has specific encryption requirements for network or IO, plea
 that the RAPIDS Accelerator suits your needs, and file and issue or discussion if you have doubts
 or would like expanded encryption support.
 
+### What is the special requirement for running Rapids Accelerator with Spark on Ray?
+RayDP provides simple APIs for running Spark on [Ray](https://github.com/ray-project/ray) and 
+integrating Spark with AI libraries, we need to make the following configuration changes to fix
+the compatibility issues with the RAPIDS Accelerator and Ray:
```

Comment:
```suggestion
[RayDP](https://github.com/oap-project/raydp) provides simple APIs for running Spark on
[Ray](https://github.com/ray-project/ray). In order to run the RAPIDS Accelerator with RayDP,
GPUs must be requested as `GPU` resources rather than `gpu` resources. This can be done
with the following changes to the typical GPU resource scheduling setup with Spark:
```